pkg/streamingpromql/benchmarks/comparison_test.go

```diff
@@ -43,6 +43,7 @@ func BenchmarkQuery(b *testing.B) {
 	cases := TestCases(MetricSizes)
 
 	opts := streamingpromql.NewTestEngineOpts()
+	opts.Timeout = 1000 * time.Second
```

Comment:
This seems like a lot :) Perhaps 60 seconds is plenty?

Reply:
This was commited by accident when I was doing other testing. Not sure if it's needed any more. Will find out when I run benchmarks later.